### DISCUSSION TITLE
Help for an old post (floating logo)

### DISCUSSION THREAD

**Discussion by Banuk1616** [templaza/astroid-framework#664] [Q&A]
Hello to all

A few weeks ago I read a post here about a floating logo. Unfortunately, I can't find this post again.

As described in that post, I recreated the floating logo on my local device. I had created a module position in the Astroid header and created a module in Joomla. It worked really well. Now the hard disk of my local device was broken and I had no backup. So I had to start from scratch on a new hard disk and now I can't do it anymore. Whenever I create the module position in the Astroid header, my main menu slides to the left and overlays the created module position for the floating logo.

Does anyone have any idea what I might be overlooking that I can't set it up again?

**Comment by Banuk1616** [comment]
Hello to all,
I was able to solve my problem. It's almost embarrassing, I had forgotten the css. Overall, I solve the floating logo problem in a completely different way now. I no longer create a module position in the Astroid header but use the logo options of Astroid. For the desired result, I added a little bit of css to my custom.css:

`.astroid-logo .astroid-logo-image {
      margin-bottom: -90px;
}`

The result is exactly how I wanted it.

**Comment by Chacapamac** [comment]
@Banuk1616 
Yes, you way is the simplest way to add a floating logo, by using directly the Astroid logo position and adding some css. 
In these 2 websites I use an custom logo module with this type of css to really float over other visual elements. The same type of css can probably work in the same manner with integrating the logo by Astroid Manager.
—> https://grafcomm.ca
—> https://efoilsurf.ca

```
div#logo-module {
  position: absolute;
  top: 50px;
  left: 150px;
}
```
